cli/src/commands/schema/get.rs

```diff
@@ -0,0 +1,53 @@
+use crate::client::{Client, GraphQLDocument};
+use crate::commands::Command;
+use crate::errors::{ErrorDetails, ExitCode, Fallible};
+use crate::telemetry::Session;
+use graphql_client::*;
+use structopt::StructOpt;
+
+#[derive(StructOpt)]
+#[structopt(rename_all = "kebab-case")]
+pub struct Get {}
+
+#[derive(GraphQLQuery)]
+#[graphql(
+    query_path = "graphql/get_schema_query.graphql",
+    schema_path = ".schema/schema.graphql",
+    response_derives = "PartialEq, Debug",
+    deprecated = "warn"
+)]
+struct GetSchemaQuery;
+
+impl Command for Get {
+    fn run(&self, session: &mut Session) -> Fallible<ExitCode> {
+        session.require_api_key()?;
+
+        let client = Client::from((session.config.api_key.as_ref().unwrap()).to_string());
```

Comment:
Future note (not this PR): might be nicer to pass in the client in to the run command, if we ever want to enable running multiple commands in the same invocation.

Reply:
Yea. I wasn't sure if we wanted to do this via session, but since we have a fair number of commands that won't need a gql client I didn't think we wanted to have to build/pass it to all commands. Don't have a strong preference.

Reply:
I didn't mean via session, just an argument, but really not an issue right now :)